mods/d2k/weapons/missiles.yaml

```diff
@@ -155,9 +155,9 @@ DeviatorMissile:
 		ExplosionPalette: deviatorgas
 		UsePlayerPalette: true
 		-ImpactSounds:
-	Warhead@4OwnerChange: ChangeOwner
+	Warhead@5OwnerChange: ChangeOwner
```

Comment:
Do we want the new `Warhead@5` moved under the new `Warhead@4`?

Reply:
I would usually say yes, but in this case i want to prioritize tagging the new release so don't want to delay this PR further.

IMO a better fix would be to have a larger followup pr that removes the number from these suffixes so that this specific regression can't happen again in the future.